### DISCUSSION TITLE
Are there any plans to support generating an openapi 3.1.x spec

### DISCUSSION THREAD

**Discussion by richardwilko** [micronaut-projects/micronaut-openapi#1222] [Ideas]
Are there any plans to support generating an openapi 3.1 spec? 

There are some nice improvements to enum models in 3.1.0. 

Thanks,
Richard

**Comment by altro3** [comment]
Base support Openapi 3.1.0 will be in 6.7.0 release. But I think, with some bugs, because I can't check all cases. You can try it and if you fiund bugs, you can create issues. I'll fix it
